src/Chapter1.hs

```diff
@@ -620,7 +636,9 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral c => c -> c
+sumLast2 n = lastDigit n + sndLastDigit n where
+  sndLastDigit n = lastDigit  ((abs n) `div` 10)
```

Comment:
Nice reusage of the `lastDigit` 🥇 

You can omit parenthesis in here as well 🙂